prow/config.yaml

```diff
@@ -4538,7 +4538,7 @@ periodics:
         defaultMode: 256
         secretName: ssh-key-secret
 
-- interval: 30m
+- interval: 6h
```

Comment:
Is this change being made for all 1.6 jobs?

Reply:
yeah, as it's getting older we'd make them run less often

Reply:
Fair enough.